### ISSUE TITLE
Idea: random redirect

### ISSUE THREAD

**Issue by anewuser** [tumpio/requestcontrol#170]
My hypothetical "random redirect" rule looks like the current redirect rule, but it always has a minimum of two `Redirect URL` fields (with the option to add/remove an nth field).

Every time a request matches the input pattern, Request Control picks one of the destinations randomly:

![random](https://user-images.githubusercontent.com/6733770/183275198-414e072f-a689-4d3a-9c02-b642e0197f40.png)

Note that this mock-up has the `Any` type, but I meant to pick `Sub document`.

See [LibRedirect](https://addons.mozilla.org/addon/libredirect/) for a list of examples where a list of possible destinations is useful.

Another example is using different image proxies to load images from third-party domains you don't want/can't connect to directly:

    https://external-content.duckduckgo.com/iu/?u=  
    https://images.weserv.nl/?url=

**Comment by ghost** [comment]
agreed, this extension could then replace LibRedirect (or farside.link with Redirector)
